sdk/textanalytics/Azure.AI.TextAnalytics/src/HealthcareOperation.cs

```diff
@@ -33,6 +33,16 @@ public class HealthcareOperation : Operation<RecognizeHealthcareEntitiesResultCo
         /// </summary>
         public override string Id { get; }
 
+        /// <summary>
+        /// Properties for healthcare operation
+        /// </summary>
+        public HealthcareOperationProperties Properties { get; set; }
+
+        /// <summary>
+        /// Gets an Status representing the operation.
+        /// </summary>
+        public JobStatus Status { get; internal set; }
```

Comment:
during the rename meeting, we agreed to call this type `TextAnalyticsOperationStatus`. Are you thinking on doing the renames in a separate PR? or address some here too

Reply:
Same comment applies for class `HealthcareOperation` which has been renamed to `AnalyzeHealthcareEntitiesOperation`
https://apiview.dev/Assemblies/Review/b196a5e53fa2441f92e3b1accefa35e7?doc=False#Azure.AI.TextAnalytics.Healthcare.AnalyzeHealthcareEntitiesOperation

Reply:
Thank you for doing the renames! You could update the Changelog in this PR to reflect the renames, or do it in a future PR